src/java.base/share/classes/sun/launcher/LauncherHelper.java

```diff
@@ -267,7 +273,7 @@ private static void printPropertyValue(String key, String value) {
                 ostream.println(s);
                 first = false;
             } else { // following lines prefix with indents
-                ostream.println(INDENT + INDENT + s);
+                ostream.println(TWOINDENT + s);
```

Comment:
I assume these changes are per reverted now.

Reply:
@AlanBateman  - I was planning to keep this trivial change. Does it not read better ?